compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELinkExecutable.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree-amd-aie/Transforms/PassDetail.h"
+#include "iree-amd-aie/Transforms/Passes.h"
+#include "iree/compiler/Codegen/Utils/LinkingUtils.h"
+#include "iree/compiler/Utils/ModuleUtils.h"
+#include "llvm/Support/FormatVariadic.h"
+#include "mlir/Pass/Pass.h"
+
+#define DEBUG_TYPE "iree-amdaie-linkexec"
```

Comment:
The `DEBUG_TYPE` should be same as the pass name -> `iree-amdaie-link-executables`.

This helps when putting in debug statements as well as quickly finding the `.cpp` as well.